### PR TITLE
Refactoring and removing hero bottom cutoff

### PR DESCRIPTION
**Pull request by danbull**
https://digitalrig.atlassian.net/browse/WEB-229

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Robots meta tags to control indexing of pages
+- WEB-229 cropping at bottom of hero
```
Comment (ThePeach) [inline CHANGELOG.md]: this should be under `Fixed`

File: src/scripts/heroAnimation/Circle.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ Circle.prototype.draw = function() {
   if (!this.active) return;
   this.ctx.beginPath();
   this.ctx.arc(this.pos.x, this.pos.y, this.radius, 0, 2 * Math.PI, false);
-  this.ctx.fillStyle = "rgba(94, 161, 184," + this.active + ")";
+  this.ctx.fillStyle = `rgba(${this.color},${this.active})`;
```
Comment (ThePeach) [inline src/scripts/heroAnimation/Circle.js]: much better :clap:

File: src/scripts/heroAnimation/hero.js [hunk 3]
```diff
@@ -119,19 +119,26 @@ function initHeader() {
...
-      var py = y + Math.random() * params.height / pointsLimiter;
-      var p = {
-        x: px,
-        originX: px,
-        y: py,
-        originY: py
-      };
-
-      params.points.push(p);
+      var px = utils.getRandomArbitrary(x, x + gridSize);
```
Comment (ThePeach) [inline src/scripts/heroAnimation/hero.js]: do we have any reason to keep these vars in a wider scope? can we reduce their pollution scope using `let` or - even better `const` ?

File: src/scripts/heroAnimation/hero.js [hunk 4]
```diff
@@ -119,19 +119,26 @@ function initHeader() {
...
+      var py = utils.getRandomArbitrary(y, y + gridSize);
+
+      if (py < params.height - gridSize / 2) {
+        var p = {
```
Comment (ThePeach) [inline src/scripts/heroAnimation/hero.js]: same as above

File: CHANGELOG.md (hunk 1 above)
Comment (danbull) [reply CHANGELOG.md]: But It's not a fix/bug it's a change due to the gradient being removed

Comment (ThePeach) [reply CHANGELOG.md]: oh

**Pull request merged**